components/document-header/template.njk

```diff
@@ -18,24 +18,30 @@
 
   {% if params.date or params.modified or params.authors or params.author %}
     <p class="app-document-header__metadata">
-      {% if params.authors %}
-        <span class="govuk-visually-hidden">Posted by: </span>
+      {%- if params.authors -%}
+        <span class="govuk-visually-hidden">Posted by</span>
         {%- for author in params.authors -%}
           {{- " and " if loop.last else (", " if not loop.first) -}}
           {{- _authorLink(author) -}}
         {%- endfor -%}
-        <span aria-hidden="true"> · </span>
-      {% elif params.author %}
-          {{- _authorLink(params.author) -}}
-          <span aria-hidden="true"> · </span>
-      {% endif %}
-      {% if params.date %}
-        <span class="govuk-visually-hidden">Posted on: </span><time datetime="{{ params.date | date }}">{{ params.date | date("d LLLL y") }}</time>
-      {% endif %}
-      {% if params.modified %}
-        <span aria-hidden="true">•</span>
-        Last updated <time datetime="{{ params.modified | date }}">{{ params.modified | date("d LLLL y") }}</time>
-      {% endif %}
+        <span aria-hidden="true">&ensp;•&ensp;</span>
+      {%- elif params.author -%}
+        {{- _authorLink(params.author) -}}
+        <span aria-hidden="true">&ensp;•&ensp;</span>
+      {%- endif -%}
+      {%- if params.date -%}
+        <span class="govuk-visually-hidden">Posted on </span><time datetime="{{ params.date | date }}">{{ params.date | date("d LLLL y") }}</time>
+      {%- endif -%}
+      {%- if params.modified -%}
+        <span aria-hidden="true">&ensp;•&ensp;</span>Last updated <time datetime="{{ params.modified | date }}">{{ params.modified | date("d LLLL y") }}</time>
+      {%- endif -%}
+      {%- if params.tags | length > 0 -%}
+        <span aria-hidden="true">&ensp;•&ensp;</span>Tags:
+        {%- for tag in params.tags %}
+          <a href="/tags/{{ tag | slug }}" class="govuk-link">{{ tag }}</a>
```

Comment:
Should we make the `/tags/` prefix configurable somehow? 🤔

Reply:
Oh, on the the tags index page I changed the link to be relative, but that won’t work within this component. There are 2 options:

Thinking about it, we might actually be able to get the URL for a tag page using `page.url`. I’ll try that.

Reply:
Have refactored everything so that any tag page that is linked to uses URL and title derived from the generated tag page. That means if you use a permalink of `/categories/{{ tag }}` for tag pages, `/categories/[tag]` will get used throughout the site.